web/compatibility.js

```diff
@@ -469,8 +469,10 @@ if (typeof PDFJS === 'undefined') {
 
   // Range requests are broken in Chrome 39 and 40, https://crbug.com/442318
   var isChromeWithRangeBug = /Chrome\/(39|40)\./.test(navigator.userAgent);
+  
+  var isWeChat = /micromessenger/.test(navigator.userAgent);
```

Comment:
It's probably best to make this regular expression case-insensitive, to prevent any future issues, e.g. change it to `/micromessenger/i.test(navigator.userAgent);` instead.